README.md

```diff
@@ -1,3 +1,96 @@
+[![Build Status](https://travis-ci.org/acquia/headless-lightning.svg?branch=master)](https://travis-ci.org/acquia/lightning)
+
 # Headless Lightning
+Headless Lightning is a more opinionated version of the [Lightning Drupal](https://github.com/acquia/lightning)
+distribution as a backend for decoupled applications.
+
+## Quick Start
+To build the codebase:
+
+    composer create-project acquia/lightning-project:dev-headless --no-interaction
+
+This will create a new directory, `lightning-project` which contains a `docroot`
+folder. This is where you should point your web server.
+
+Just like any Drupal project, you will need a working [LAMP stack](https://www.drupal.org/node/2461571)
+and an available database within that stack. Once you have that setup, you can
+use Drush, DrupalConsole, or the web interface to install the site.
+
+**There is no need to clone or fork this project unless you want to contribute
```

Comment:
Do we need this paragraph? We already delivered the installation command, so maybe this is redundant?